airflow/providers/amazon/aws/transfers/mysql_to_s3.py

```diff
@@ -105,7 +105,8 @@ def _fix_int_dtypes(self, df):
                 notna_series = df[col].dropna().values
                 if np.isclose(notna_series, notna_series.astype(int)).all():
                     # set to dtype that retains integers and supports NaNs
-                    df[col] = np.where(df[col].isnull(), None, df[col]).astype(pd.Int64Dtype)
+                    df[col] = np.where(df[col].isnull(), None, df[col])
+                    df[col] = df[col].astype(pd.Int64Dtype())
```

Comment:
Don't we have a test for that? Maybe can you add one to see how it should look like after this?

Reply:
The test for this operator **doesn't** cover this method (`_fix_int_dtypes`). So I'll add one.

Thanks for the input!

Reply:
@gmontanola are you planning to finish it?

Reply:
I totally forgot this. Well, I guess adding this test should be pretty straight forward. I'll try to add this this week. Are you willing to add this by yourself or found a better a way to fix the problem?